support/arcade/mra_loader.cpp

```diff
@@ -649,11 +649,13 @@ static int xml_send_rom(XMLEvent evt, const XMLNode* node, SXML_CHAR* text, cons
 		{
 		int result = buffer_append(arc_info->data, text);
 		if (result<0)
+			{
```

Comment:
Perhaps you should also fix the indentation (the 2 lines above and everything inside the new scope you created). Right now it is not consistent with the indentation in the rest of the file.

Reply:
the ugly mix of tabs and spaces should be fixed now.